src/migration-scripts/firewall/16-to-17

```diff
@@ -56,5 +59,4 @@ def migrate(config: ConfigTree) -> None:
 
         for base_hook in [['forward', 'filter'], ['input', 'filter'], ['prerouting', 'raw']]:
             tmp_path = firewall_base + [family] + base_hook
-            if config.exists(tmp_path):
```

Comment:
Why remove the conditional?

Reply:
It's checked on entry to migrate_chain(), not much of a saving. Both call sites need the check, putting it inside the function made sense.  

The other call site needs to check if the config exists before looping over named sub-nodes, though.